CONTRIBUTING을 위한 make check, make testall 오류

안녕하세요.
[CONTRIBUTING](https://github.com/konlpy/konlpy/blob/master/CONTRIBUTING.rst#contributing)을 보고, **3. 이슈 제안/해결하기**에 오류가 있습니다.
제가 파악한 내용입니다.

_python3.7에서 환경설정 후 실행했습니다._

make check 실행시
```
$ make check

check-manifest
lists of files in version control and sdist do not match!
missing from sdist:
  .gitmodules
  CONTRIBUTING-en.rst
  CONTRIBUTING.rst
  LICENSE
  Makefile
  build.xml
  docs/Makefile
  docs/_static/bootstrap-affix.min.js
  docs/_static/jquery-1.11.2.min.js
  docs/_static/konlpy.css
  docs/_static/konlpy.png
  docs/_templates/github-banner.html
  docs/_templates/github.html
  docs/_templates/page.html
  docs/_templates/sidebar-footer.html
  docs/_templates/sidebar-header.html
  docs/_templates/translations.html
  docs/_themes
  docs/api.rst
  docs/api/konlpy.corpus.rst
  docs/api/konlpy.rst
  docs/api/konlpy.tag.rst
  docs/changelog.rst
  docs/conf.py
  docs/data.rst
  docs/examples.rst
  docs/examples/babel.png
  docs/examples/chunking.png
  docs/examples/chunking.py
  docs/examples/chunking.rst
  docs/examples/chunking_thumb.png
  docs/examples/collocations.py
  docs/examples/collocations.rst
  docs/examples/corpus.py
  docs/examples/corpus.rst
  docs/examples/explore.png
  docs/examples/explore.py
  docs/examples/explore.rst
  docs/examples/generate.py
  docs/examples/generate.rst
  docs/examples/heap-1000.png
  docs/examples/heap.png
  docs/examples/heap_thumb.png
  docs/examples/multithreading.png
  docs/examples/multithreading.py
  docs/examples/multithreading.rst
  docs/examples/shoes.png
  docs/examples/wordcloud.png
  docs/examples/wordcloud.py
  docs/examples/wordcloud.rst
  docs/examples/zipf.png
  docs/images/python-bits.png
  docs/images/time.png
  docs/images/windows-bits.png
  docs/index.rst
  docs/install.rst
  docs/locale/ko/LC_MESSAGES/_templates.po
  docs/locale/ko/LC_MESSAGES/_themes.po
  docs/locale/ko/LC_MESSAGES/_themes/README.po
  docs/locale/ko/LC_MESSAGES/api.po
  docs/locale/ko/LC_MESSAGES/api/konlpy.corpus.po
  docs/locale/ko/LC_MESSAGES/api/konlpy.po
  docs/locale/ko/LC_MESSAGES/api/konlpy.tag.po
  docs/locale/ko/LC_MESSAGES/changelog.po
  docs/locale/ko/LC_MESSAGES/data.po
  docs/locale/ko/LC_MESSAGES/examples.po
  docs/locale/ko/LC_MESSAGES/examples/collocations.po
  docs/locale/ko/LC_MESSAGES/examples/explore.po
  docs/locale/ko/LC_MESSAGES/examples/sentiment.po
  docs/locale/ko/LC_MESSAGES/examples/stopwords.po
  docs/locale/ko/LC_MESSAGES/examples/wordcloud.po
  docs/locale/ko/LC_MESSAGES/faq.po
  docs/locale/ko/LC_MESSAGES/index.po
  docs/locale/ko/LC_MESSAGES/install.po
  docs/locale/ko/LC_MESSAGES/morph.po
  docs/locale/ko/LC_MESSAGES/nlp.po
  docs/locale/ko/LC_MESSAGES/references.po
  docs/locale/ko/LC_MESSAGES/start.po
  docs/locale/ko/LC_MESSAGES/test.po
  docs/make.bat
  docs/morph-0.csv
  docs/morph-1.csv
  docs/morph-2.csv
  docs/morph-3.csv
  docs/morph.csv
  docs/morph.py
  docs/morph.rst
  docs/references.rst
  docs/start.rst
  docs/test.rst
  konlpy/java/src/kr/lucypark/jhannanum/comm/HannanumInterface.java
  konlpy/java/src/kr/lucypark/jhannanum/hannanum/WorkflowFactory.java
  konlpy/java/src/kr/lucypark/kkma/KkmaInterface.java
  konlpy/java/src/kr/lucypark/komoran/KomoranInterface.java
  konlpy/java/src/kr/lucypark/okt/OktInterface.java
  pypirc.sample
  reinstall
  requirements-dev.txt
  scripts/Dockerfile.centos:7
  scripts/Dockerfile.centos:8
  scripts/Dockerfile.template:centos
  scripts/Dockerfile.template:ubuntu
  scripts/Dockerfile.ubuntu:bionic
  scripts/Dockerfile.ubuntu:disco
  scripts/Dockerfile.ubuntu:eoan
  scripts/Dockerfile.ubuntu:xenial
  scripts/Dockerfile.ubuntu:xenial-testpypi
  scripts/Dockerfile.windows:10
  scripts/mecab.sh
suggested MANIFEST.in rules:
  include *.rst
  include *.sample
  include *.txt
  include *.xml
  include LICENSE
  include Makefile
  include reinstall
  recursive-include docs *.bat
  recursive-include docs *.css
  recursive-include docs *.csv
  recursive-include docs *.html
  recursive-include docs *.js
  recursive-include docs *.png
  recursive-include docs *.po
  recursive-include docs *.py
  recursive-include docs *.rst
  recursive-include docs Makefile
  recursive-include konlpy *.java
  recursive-include scripts *.sh
you have docs/locale/ko/LC_MESSAGES/_templates.mo in source control!
that's a bad idea: auto-generated files should not be versioned
this also applies to the following:
  docs/locale/ko/LC_MESSAGES/_themes.mo
  docs/locale/ko/LC_MESSAGES/_themes/README.mo
  docs/locale/ko/LC_MESSAGES/api.mo
  docs/locale/ko/LC_MESSAGES/api/konlpy.corpus.mo
  docs/locale/ko/LC_MESSAGES/api/konlpy.mo
  docs/locale/ko/LC_MESSAGES/api/konlpy.tag.mo
  docs/locale/ko/LC_MESSAGES/changelog.mo
  docs/locale/ko/LC_MESSAGES/data.mo
  docs/locale/ko/LC_MESSAGES/examples.mo
  docs/locale/ko/LC_MESSAGES/examples/collocations.mo
  docs/locale/ko/LC_MESSAGES/examples/explore.mo
  docs/locale/ko/LC_MESSAGES/examples/sentiment.mo
  docs/locale/ko/LC_MESSAGES/examples/stopwords.mo
  docs/locale/ko/LC_MESSAGES/examples/wordcloud.mo
  docs/locale/ko/LC_MESSAGES/faq.mo
  docs/locale/ko/LC_MESSAGES/index.mo
  docs/locale/ko/LC_MESSAGES/install.mo
  docs/locale/ko/LC_MESSAGES/morph.mo
  docs/locale/ko/LC_MESSAGES/nlp.mo
  docs/locale/ko/LC_MESSAGES/references.mo
  docs/locale/ko/LC_MESSAGES/start.mo
  docs/locale/ko/LC_MESSAGES/test.mo
make: *** [check] Error 1
```
- 어떤 메세지인지 파악이 안되서 오류 내용만 남겨놓습니다.

make testall 실행시
```
...
$ make testall
...
========================================================================================= short test summary info ==========================================================================================
FAILED test/test_stream_daum.py::test_daum_streamer - AttributeError: 'NoneType' object has no attribute 'findAll'
FAILED test/test_stream_naver.py::test_naver_streamer - AttributeError: 'NoneType' object has no attribute 'findAll'
================================================================================ 2 failed, 38 passed, 2 warnings in 54.23s =================================================================================
make: *** [testall] Error 1
```

```
FAILED test/test_stream_daum.py

from konlpy.stream import daum
streamer = daum.DaumStreamer()
streamer.stream()
```

- 다음에서 실검 제공을 하지 않아, get_current_trend() 호출할 때 오류 발생
  - 참고: https://www.nocutnews.co.kr/news/5262704

```
FAILED test/test_stream_naver.py

>>> from konlpy.stream import naver
>>> streamer = naver.NaverStreamer()
>>> streamer.stream()
```
- 네이버 첫 화면에 실검이 없어, get_current_trend() 호출할 때 오류 발생

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1,4 +1,4 @@
-include requirements.txt
+include requirements.txt requirements2.txt
 recursive-include konlpy/data *
 recursive-include konlpy/java *.class *.jar *.json
 recursive-include konlpy/java/data *

--- a/Makefile
+++ b/Makefile
@@ -22,8 +22,13 @@ check:
 	--ignore-bad-ideas *.mo
 
 	pyroma dist/konlpy-*tar.gz
-	pep8 --ignore=E501 konlpy/*.py
-	pep8 --ignore=E501 konlpy/*/*.py
+
+	# E126: Continuation line over-indented
+	# E402: Module level import not at top of file
+	# E501: Line too long
+	# E701: Multiple statements on one line(colon)
+	pep8 --ignore=E501,E402 konlpy/*.py
+	pep8 --ignore=E501,E701,E126 konlpy/*/*.py
 
 testpypi:
 	python setup.py register -r pypitest

--- a/Makefile
+++ b/Makefile
@@ -15,7 +15,12 @@ build:
 	python setup.py sdist --formats=gztar,zip
 
 check:
-	check-manifest
+	check-manifest \
+	--ignore \
+	.gitmodules,CONTRIBUTING*,LICENSE,Makefile,build.xml,pypirc.sample,reinstall,requirements-dev.txt,\
+	docs/**,konlpy/java/src/**,scripts/** \
+	--ignore-bad-ideas *.mo
+
 	pyroma dist/konlpy-*tar.gz
 	pep8 --ignore=E501 konlpy/*.py
 	pep8 --ignore=E501 konlpy/*/*.py

--- a/konlpy/data.py
+++ b/konlpy/data.py
@@ -207,6 +207,6 @@ class StringWriter(object):
 
 
 __all__ = [
-    'find', 'load', 'listdir', 'clear', 
+    'find', 'load', 'listdir', 'clear',
     'path', 'FileSystemPathPointer', 'PathPointer',
-    'CorpusReader', 'StringWriter',]
+    'CorpusReader', 'StringWriter']

--- a/konlpy/tag/_okt.py
+++ b/konlpy/tag/_okt.py
@@ -92,7 +92,7 @@ class Okt():
         """Phrase extractor."""
 
         return [p for p in self.jki.phrases(phrase).toArray()]
-    
+
     def normalize(self, phrase):
         text = self.jki.normalize(phrase)
         return text

